.github/workflows/build.yml

```diff
@@ -41,4 +42,4 @@ jobs:
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
       - name: "Test comand ${{ matrix.nixCommand }}"
-        run: "${{ matrix.nixCommand }} -L --show-trace"
+        run: "${{ matrix.nixCommand }} -L --show-trace --system ${{ matrix.platform.system }}"
```

Comment:
```suggestion
        run: "nix build .#holochain --no-link -L --show-trace --system ${{ matrix.platform.system }}"
```

Reply:
is this cross-compilation or using apple's compatibility mode?

Reply:
It is using the compatibility mode, as I only set system. For cross compiling I would have to set `crossSystem` which is not a CLI argument but a nixpkgs import argument.